apps/camera_pipe/camera_pipe.cpp

```diff
@@ -2,6 +2,8 @@
 #include <stdint.h>
 
 using namespace Halide;
+using namespace Halide::Internal;
+IRPrinter irp(std::cerr);
```

Comment:
Yes.  It was used for interactively debugging the app with lldb.  I will remove this.